packages/grafana-prometheus/package.json

```diff
@@ -24,10 +24,52 @@
     "./LICENSE_AGPL"
   ],
   "publishConfig": {
+    "access": "public",
     "main": "./dist/cjs/index.cjs",
     "module": "./dist/esm/index.mjs",
     "types": "./dist/cjs/index.d.cts",
-    "access": "public"
+    "exports": {
```

Comment:
After these, will this https://github.com/grafana/grafana-amazonprometheus-datasource/blob/main/src/module.ts still work? Do we need a default export?

Reply:
Great question - I'll attempt to validate this.

Reply:
I did some testing. In the Amazon Prometheus DS project, I started by hard-copying the updated `@grafana/prometheus`' `dist` into `node_modules`:

```
rm -rf node_modules/@grafana/prometheus/dist
cp -R /PATH/TO/grafana/packages/grafana-prometheus/dist node_modules/@grafana/prometheus/dist
```

The unit tests passed (with `yarn run test:ci`. The e2e tests didn't pass before or after hard-copying the updated `@grafana/prometheus`' `dist` into `node_modules`.

I believe this points to these updates working as expected in the Amazon Prometheus DS :)

cc: @grafana/enterprise-datasources